addons/website_slides/controllers/main.py

```diff
@@ -321,6 +325,18 @@ def create_slide(self, *args, **post):
             return {'error': _('Internal server error, please try again later or contact administrator.\nHere is the error message: %s') % e.message}
         return {'url': "/slides/slide/%s" % (slide_id.id)}
 
+    @http.route("/slides/max_size", type="json", auth="public", website=True)
```

Comment:
I tried with JSONRPC client-side, but public users cannot have access to `ir.confir_parameter` objects (that's why I use `sudo()` below), which is logical BTW. If you have a better idea, I'm open to it...